support_branding/views/res_config_settings.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="support_res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">support.res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="10" />
+        <field name="inherit_id" ref="base.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('settings')]" position="inside">
+                <div
+                    class="app_settings_block"
+                    data-string="Support Settings"
+                    string="Support Settings"
+                    data-key="support_company"
```

Comment:
Found a small thing:

![image](https://user-images.githubusercontent.com/1466356/160192178-cab8ff93-753f-4a31-a775-074ae92e8136.png)

in Enterprise, this menu doesnt have an Icon. 

I think we can use the `static/description/icon.png` but I also saw that it had been changed back to the default whereas it was this cool lifeboat icon - could we get back that icon and also use it here?

Reply:
Tried this but it had no effect, it was still blank. Also OCA recommends we use there default icon, so not sure how flexible are the rules.

Reply:
Ok lets use the OCA icon for now